mne_bids/write.py

```diff
@@ -1543,8 +1589,77 @@ def write_raw_bids(raw, bids_path, events_data=None,
     return bids_path
 
 
-def write_anat(image, bids_path, raw=None, trans=None, landmarks=None,
-               t1w=None, deface=False, overwrite=False, verbose=False):
+def get_anat_landmarks(image, info, trans, fs_subject, fs_subjects_dir=None):
+    """Get landmarks transformed to image voxel coordinates from head space.
+
+    Parameters
+    ----------
+    image : str | pathlib.Path | NibabelImageObject
```

Comment:
should also accept a `BIDSPath`, no? And if it does, we can allow for `fs_subject=None` and get `subject` from `image.subject`?
@agramfort WDYT?

Reply:
yes !

Reply:
in fact not. You call this before calling write_anat so the bids_path you have at this stage is not pointing to the MRI on disk. I would keep it as it is.

Reply:
That was my thought process as well